### DISCUSSION TITLE
Experiment description

### DISCUSSION THREAD

**Discussion by mrbazzik** [mlflow/mlflow#6765] [Q&A] [answered]
Looking through docs, I can't find a way to provide `description` when I create a new experiment, either via `mlflow` or `mlflow.client` API. I see `description` parameter when creating models, model versions, runs, but not experiments. Is there a way to do that?

**Comment by JHibbard** [comment, marked as answer]
Great question, the MLflow UI uses the "mlflow.note.content" system tag for populating the experiment's description. You can use the normal create_experiment method on the client like so:

```python
from  mlflow.tracking import MlflowClient

client = MlflowClient()
client.create_experiment(
    name='Example', 
    tags={"mlflow.note.content": "My experiment description..."}
)
```

You can find a list of the other reserved system tags [here](https://mlflow.org/docs/latest/tracking.html#system-tags) in the docs. Hope that helps!

**Reply by mrbazzik** [reply]
Thanks for the pointer! 
On the page, following the link, it says: 
> A descriptive note about this run. This reserved tag is not set automatically and can be overridden by the user to include additional information about the run. The content is displayed on the run’s page under the Notes section.

I'd say it's not completely clear from this description that it'll populate the experiment's description rather than a particular run's metadata.
